tensorflow/core/kernels/matmul_op_test.cc

```diff
@@ -22,9 +22,9 @@ namespace tensorflow {
 
 static Graph* Matmul(int m, int k, int n, bool transpose_a, bool transpose_b) {
   Graph* g = new Graph(OpRegistry::Global());
-  Tensor in0(DT_FLOAT, transpose_a ? TensorShape({k, m}) : TensorShape({m, k}));
+  Tensor in0(DT_COMPLEX64, transpose_a ? TensorShape({k, m}) : TensorShape({m, k}));
```

Comment:
I suspect you don't want to do this, this is changing all of the tests from float to complex, instead of adding complex to the existing suite